diff/basicAuth.go

```diff
@@ -94,8 +96,9 @@ func (sc *Syncer) createUpdateBasicAuth(basicAuth *state.BasicAuth) (*Event, err
 		}, nil
 	}
 	if err != nil {
-		return nil, errors.Wrapf(err, "error looking up basic-auth %v",
-			*basicAuth.Username)
+		return nil, fmt.Errorf("error looking up basic-auth %v: %w",
```

Comment:
```suggestion
		return nil, fmt.Errorf("error looking up basic-auth %q: %w",
```

same consistency note as for aclGroup